elements/bulbs-reading-list/lib/reading-list-article.js

```diff
@@ -106,8 +106,39 @@ export default class ReadingListArticle {
     this.element.dataset.loadStatus = 'loading';
   }
 
+  setupTaboola () {
+    let taboola = window._taboola || [];
```

Comment:
Should this stash the new array on `window._taboola` if `window._taboola` undefined so the taboola script can find it later?